Lusas_Adapter/CRUD/Create/Create.cs

```diff
@@ -374,6 +376,53 @@ private bool CreateCollection(IEnumerable<Edge> edges)
 
         /***************************************************/
 
+        private bool CreateCollection(IEnumerable<Opening> openings)
+        {
+            if (openings != null)
+            {
+                CreateTags(openings);
+
+                List<Opening> validOpenings = new List<Opening>();
+
+                foreach (Opening opening in openings)
+                {
+                    if (CheckPropertyError(opening, p => p.Edges))
+                        if (CheckPropertyError(opening.Edges, e => e.Select(x => x.Curve)))
+                        {
+                            if (opening.Edges.All(x => !Engine.Adapters.Lusas.Query.InvalidEdge(x)))
+                            {
+                                if (Engine.Spatial.Query.IsPlanar(opening, false, Tolerance.MacroDistance)) //Check if this works.
```

Comment:
```suggestion
                                if (Engine.Spatial.Query.IsPlanar(opening, false, m_mergeTolerance))
```

Did we not have a merge tolerance from the settings? Would it not be worth using that as it would be the same tolerance used in Lusas I imagine. Could be something we test in the testing sprint.